frontend/app/(authenticated)/question/[id]/page.tsx

```diff
@@ -32,7 +32,7 @@ export default function QuestionPage() {
     refetch();
   };
 
-  const isOpenQuestion = !question?.replierId;
+  const isOpenQuestion = !question?.replierId || !profile.hasLaunchedKeys;
```

Comment:
I believe this condition is incorrect. If user hasn't launched his keys, it doesn't make it an open-question.
An open-question is special kind of question that doesn't have a target user. Any user can reply to it

Reply:
Agreed, but do you think we should still show the comment list below in that component if profile didn't launch a key?

Reply:
I'm not sure I understand you question, but:

Comment list should always be visible for open questions.
The content of the comment itself is removed by the backend if the user doesn't have a key.

For regular questions, comments should be visible if the question has been replied to.
If the question is not replied, it's not possible to add any comment.

Reply:
"The content of the comment itself is removed by the backend if the user doesn't have a key."

But now we may want to allow showing the comment if the receiving user didn't launch a key I guess